### ISSUE TITLE
Not getting proper mask as instance.

### ISSUE THREAD

**Issue by vivekdeepquanty** [pytorch/vision#2108]
Hi guys

Use pretrained weights=yes
no. of epoch=400
no. of class=1

At the time of prediction i am not getting individual masks for individual object. i am getting some extra mask but those are empty or partial . what can be the reason?

**Comment by fmassa** [comment]
Hi,

There might be many reasons, but without further information it is difficult to be 100% sure.
- Maybe your training dataset is too small
- Maybe there is a bug in the code
- Maybe the annotations in the dataset are too noisy

As this is a very open question, I'm closing it, but please let us know if you find issues with torchvision.
